bootstrap.sh

```diff
@@ -108,14 +106,12 @@ tar zxf logstash-2.2.1.tar.gz
 ln -sf logstash-2.2.1 logstash
 
 echo "Setting up init script for dmon-ls ..."
-cp $DMONHOME/src/init/dmon-ls /etc/init.d/dmon-ls
-chmod +x /etc/init.d/dmon-ls
-update-rc.d dmon-ls defaults 96 9
+cp $DMONHOME/src/init/dmon-ls.conf /etc/init/dmon-ls.conf
 
 #Setup Logrotate
 echo "Setting up logrotate ..."
 
-echo "/opt/IeAT-DICE-Repository/src/logs/logstash.log{
+echo "/var/log/upstart/dmon-ls.log{
```

Comment:
Logstash logs are stored in /opt/DICE-Monitoring/src/logs/logstash.log (upstart script passes "-l /opt/DICE-Monitoring/src/logs/logstash.log" flag to logstash agent command), upstart logs only contain standard output and error from logstash command, which should be mostly empty.

Reply:
Thanks for spotting this. I was thinking about elasticsearch, which does not have the redirect of log.